chromium-headless/verify.py

```diff
@@ -0,0 +1,14 @@
+import subprocess
+import pychrome
+
+# verify the google-chrome and chromedriver have the same version (exluding patch level)
```

Comment:
can we move this check to chromium? relevant for there too. (then dont need to add it here since we dont reinstall)

Reply:
[Fixed](https://github.com/demisto/dockerfiles/pull/21881/commits/bd7fedf4d0dd0a69b3fd9d1bff5ebb3cc726f8fd).